docs/QueryManagement.md

```diff
@@ -0,0 +1,104 @@
+<a name="Query Logging Management"></a>
+
+Functionality Overview
+======================
+
+Query Logging Management is an addition to TorQ to enhance the current query logging system. The aim for this tool is to have access to information about queries that are sent to specific processes throughout the day, allowing for query analysis to be carried. 
+
+For each query executed we want access to:
+
+- The time of the query
+- The amount of time the query took to run
+- Username of the person running the query
+- IP address of the person running the query
+- Host of the process that was being queried 
+- Name of the process that was being queried 
+- Process type of the process that was being queried 
+- The query that was being executed including any parameters 
+
+This query logging functionality can be enabled or disabled for specific processes using variables within TorQ config files.
+
+Architecture
+============
+
+The architecture of the Query Logging Management framework is shown in the following diagram:
+
+![QueryLoggingManagementArchitecture](graphics/torq-qlm_architecture.PNG)
+
+Processes
+========
+
+Query Feed
+----------
+
+If enabled within a TorQ process, the Query Feed process subscribes to updates from the .usage.usage table that is defined in the logusage.q handlers script from all enabled TorQ processes. 
```

Comment:
I'm not sure "If enabled within a TorQ process" is correct - should this be "within a TorQ stack"?